model/docs/example_with_datasets.qmd

```diff
@@ -297,21 +297,20 @@ We can also take a look at the latent infections:
 out2 = hosp_model.plot_posterior(var="latent_infections", ylab="Latent Infections")
 ```
 
-## Round 2: Incorporating weekday effects
+## Round 2: Incorporating day-of-the-week effects
```

Comment:
Can we use the day of week effect provided by process, rather than writing our own here?

Reply:
We could, but I also like having more examples extending the module. Perhaps, we could simplify this by avoiding the unnecessary callbacks (as we discussed today?)

Reply:
Resolving this by linking to the built in class above, but keeping the code here.